setup.py

```diff
@@ -3,13 +3,21 @@
 """The setup script."""
 
 from setuptools import find_packages, setup
+import os
 
 with open("README.rst") as readme_file:
     readme = readme_file.read()
 
 with open("HISTORY.rst") as history_file:
     history = history_file.read()
 
+# extract __version__ from __init__.py
+_here = os.path.abspath(os.path.dirname(__file__))
+_version = {}
+with open(os.path.join(_here, 'parselglossy', '__init__.py')) as f:
```

Comment:
Not terribly important, but could you do it with `pathlib`? :smile_cat:
```with Path("parselglossy/__init__.py").open("r") as f:
          exec(f.read(), _version)
```

Reply:
I was thinking about it and was wondering whether this would somehow limit us but I think it is a good idea. I will modernize that.